gcloud_dataproc/v02/create_cluster_without_VEP.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
copied back from previously deleted file, cleaned up to  have a helper `create_cluster ` function as well as the main, and updated some of the args to be up-to-date with latest image versions, hail init code, etc